client/src/App.js

```diff
@@ -22,6 +25,9 @@ function App() {
             <Route path="/signup" component={SignUp} />
             <Route path="/login" component={Login} />
             <Route path="/" exact component={SignUp} />
+            <Route path="/profile" component={Profile} />
+            <Route path="/reviews" component={Reviews} />
+            <Route path="/balance" component={Balance} />
```

Comment:
These should be Protected - I thought we merged that PR into dev @sorrelbri ?

Reply:
I might have used an older version of dev before Sorrel merge that PR

Reply:
Merged this morning: #32